index.js

```diff
@@ -42,15 +42,16 @@ class GestureRecognizer extends Component {
       onPanResponderTerminate: responderEnd
     });
   }
-  
+
   componentDidUpdate(prevProps) {
     if (this.props.config !== prevProps.config) {
       this.swipeConfig = Object.assign(swipeConfig, this.props.config);
     }
   }
-  
+
   _handleShouldSetPanResponder(evt, gestureState) {
     return (
+      this.props.swipeEnabled &&
```

Comment:
Rather than setting a `defaultProp` value below, you could simply check if swipeEnabled is not false:
`this.props.swipeEnabled !== false`